rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -413,6 +413,90 @@ s32 cellSysutilGameReboot_I()
 	fmt::throw_exception("Unimplemented" HERE);
 }
 
+s32 _ZN4cxml7Element11AppendChildERS0_()
```

Comment:
They can, but I'll do it later. We still need mangled names to generate correct function IDs btw.